baseframe/templates/baseframe/mui/forms.html.jinja2

```diff
@@ -193,6 +197,9 @@
   <script type="text/javascript">
     $(function() {
   {%- endif %}
+      if (typeof(tinyMCE) != 'undefined') {
+        tinymce.remove('textarea#{{ field.id }}');
+      }
```

Comment:
Why was this change needed?

Reply:
Tinymce was failing to load the second time since there isn't a page reload in case of hgtv spa.
So, this line takes care of cleanly removing any previous instance tinymce.